examples/embed/spectrogram/spectrogram.py

```diff
@@ -171,14 +156,39 @@ def make_spectrogram():
         inner_radius="inner_radius", outer_radius="outer_radius",
         start_angle="start_angle", end_angle="end_angle",
         source=radial_source, name="eq")
-    eq.grid.grid_line_color=None
+    eq.grid.grid_line_color = None
+
+    return freq_slider, gain_slider, spectrum, signal, spec, eq
+
+
+def get_layout():
```

Comment:
This is unused, right? Let's just delete it, I think leaving it would be confusing.